flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/TableFunction.scala

```diff
@@ -131,8 +202,10 @@ abstract class TableFunction[T] extends UserDefinedFunction {
     * method. Flink's type extraction facilities can handle basic types or
```

Comment:
I think the comment should also be updated, to make user be aware of the behavior that he can determine the result type based on the parameters.